ddl/db_integration_test.go

```diff
@@ -2082,3 +2082,14 @@ func (s *testIntegrationSuite3) TestSqlFunctionsInGeneratedColumns(c *C) {
 	tk.MustGetErrCode("create table t (a int, b int as (statement_digest(1)))", mysql.ErrGeneratedColumnFunctionIsNotAllowed)
 	tk.MustGetErrCode("create table t (a int, b int as (statement_digest_text(1)))", mysql.ErrGeneratedColumnFunctionIsNotAllowed)
 }
+
+func (s *testIntegrationSuite3) TestParserIssue284(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("create table test.t_parser_issue_284(c1 int not null primary key)")
```

Comment:
Table `t_parser_issue_284 ` created but not used? Should used when to create table `t_parser_issue_284_2 `?

Reply:
Sure, [fixed](https://github.com/pingcap/tidb/pull/12548/files#diff-703ae6b7872b425273d1832c198598c8R2090)